charts/agent/templates/daemonset-windows.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+#  name: {{ include "agent-windows.fullname" . }}
+  name: {{ include "agent.fullname" . | printf "%s-windows" }}
+  labels:
+    {{- include "agent-windows.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "agent-windows.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "agent-windows.labels" . | nindent 8 }}
+        {{- with .Values.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        windowsOptions:
+          hostProcess: true
+          runAsUserName: "NT AUTHORITY\\SYSTEM"
+      hostNetwork: true
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.windowsImage.repository }}:{{ .Values.windowsImage.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.windowsImage.pullPolicy | default .Values.global.image.pullPolicy }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+          - name: config
+            mountPath: /Sysdig/Agent/Config
+          - name: secrets
+            mountPath: /Sysdig/Agent/Secrets
+      terminationGracePeriodSeconds: 0
+      volumes:
+      - name: config
+        configMap:
+          name: {{ include "agent.configmapName" . }}
+      - name: secrets
+        secret:
+          {{- if not ( include "agent.accessKeySecret" . ) }}
+          secretName: {{ template "agent.fullname" . }}
+          {{- else }}
+          secretName: {{ include "agent.accessKeySecret" . }}
+          {{- end }}
+      nodeSelector:
```

Comment:
@iurly would it make sense to add
```yaml
"kubernetes.io/os": linux
```
in:
- charts/agent/templates/daemonset.yaml
- charts/agent/templates/deployment.yaml
?